xtra-libp2p/src/endpoint.rs

```diff
@@ -534,6 +535,8 @@ impl Endpoint {
 
 impl Endpoint {
     async fn notify_connection_established(&mut self, peer: PeerId) {
+        tracing::info!(%peer, "New connection established");
```

Comment:
nit: it will be easier grep for these if they look similar when establishing/dropping. 

```suggestion
        tracing::info!(%peer, "Connection established");
```